### PR TITLE
Add Deno usage instructions in README

### PR DESCRIPTION
**Pull request by anniel-stripe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,6 +152,21 @@ stripe.customers
       });
   });
 ```
+### Usage with Deno
+
+As of 11.16.0, stripe-node provides a `deno` export target. You may import stripe-node with Deno through npm specifiers or a third party CDN like esm.sh.
```
Comment (anniel-stripe) [inline README.md:157]:
```suggestion
As of 11.16.0, stripe-node provides a `deno` export target. You may import stripe-node in Deno through npm specifiers or a third party CDN like esm.sh.
```

File: README.md [hunk 2]
```diff
@@ -152,6 +152,21 @@ stripe.customers
...
+```js
+import Stripe from 'npm:stripe';
+```
+
+Import using esm.sh:
+```js
+import Stripe from 'https://esm.sh/stripe?target=deno';
+```
+
+Please see https://github.com/stripe-samples/stripe-node-deno-samples for more detailed examples and instructions on how to use stripe-node with Deno.
```
Comment (anniel-stripe) [inline README.md:169]:
```suggestion
Please see https://github.com/stripe-samples/stripe-node-deno-samples for more detailed examples and instructions on how to use stripe-node in Deno.
```

File: README.md [hunk 3]
```diff
@@ -152,6 +152,21 @@ stripe.customers
...
+### Usage with Deno
+
+As of 11.16.0, stripe-node provides a `deno` export target. You may import stripe-node in Deno through npm specifiers or a third party CDN like esm.sh.
+
+Import using npm specifiers:
+```js
+import Stripe from 'npm:stripe';
+```
+
+Import using esm.sh:
```
Comment (pakrym-stripe) [inline README.md:164]: I think we should encourage `npm:stripe` as a safer alternative to having library re-processed via a third party service.

**Review by pakrym-stripe — CHANGES_REQUESTED** [review]
Let's only keep the `npm:stripe` version.

File: README.md (hunk 3 above)
Comment (anniel-stripe) [reply README.md]: I removed the references to esm.sh, if a user really wants to use a CDN they can refer to the Deno docs

**Review by pakrym-stripe — APPROVED** [review]

**Pull request merged**